src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @flow
+ * @file Preview sidebar additional tab component
```

Comment:
These aren't strictly tied to Preview, so should this be something like `Sidebar Additional Tab component`?